src/lib.rs

```diff
@@ -685,6 +684,34 @@ impl Uuid {
         Uuid { bytes: b }
     }
 
+    /// Creates a v4 Uuid from rng bytes (ref issue #170)
```

Comment:
Probably don't need to reference GitHub issues in the docs prose.

Reply:
sure. Since we have so many confusing functions to generate a UUID wanted to leave a note on why do we need this one :smile: will remove it